src/cc_catalog_airflow/dags/util/config.py

```diff
@@ -34,15 +34,19 @@
 DAG_VARIABLES = {
     'flickr': {
         SCRIPT: os.path.join(API_SCRIPT_PATH, 'Flickr.py'),
+        CRONTAB_STR: '0 * * * *'
```

Comment:
It runs every hour so that it only has to process one hour of data at a time.  I agree that it's too often, though, especially since in the long run we need to regularly reprocess data from each provider to keep it fresh.  By Friday, I'm going to have an issue written to overhaul that script with an eye towards performance, with the hope that we can do daily runs of that script instead of hourly.

Reply:
If I remember correctly, the Flickr API job also has date parameters that process data within a time period, not their full catalog, but correct me if I'm wrong @mathemancer

Reply:
@brenoferreira That is correct.  It's not feasible to process the entire Flickr catalog using our current methods, so the hourly job processes only the data produced within the previous hour.